istioctl/pkg/install/k8sversion/version.go

```diff
@@ -31,7 +32,7 @@ const (
 	MinK8SVersion               = 26
 	UnSupportedK8SVersionLogMsg = "\nThe Kubernetes version %s is not supported by Istio %s. The minimum supported Kubernetes version is 1.%d.\n" +
 		"Proceeding with the installation, but you might experience problems. " +
-		"See https://istio.io/latest/docs/setup/platform-setup/ for a list of supported versions.\n"
+		"See https://istio.io/%s/docs/setup/platform-setup/ for a list of supported versions.\n"
```

Comment:
IMO pointing to the latest is fine, the old docs are archieved, which should not be used for reference.